### DISCUSSION TITLE
Pre-Flashed Tasmota Devices for Sale?

### DISCUSSION THREAD

**Discussion by tdampier** [arendst/Tasmota#15637] [Q&A] [answered]
Looking for a tasmota pre-flashed devices to be used in 3 ceiling fans with no remotes and no switch wiring but pull-chain.  Would like to integrate with smartthings with full integration.

Does anyone sell tasmota pre-flashed devices that I can purchase?

**Comment by barbudor** [comment, marked as answer]
Some are listed here https://templates.blakadder.com/preflashed.html

Their might be more

**Comment by tdampier** [comment]
And none of them control fans/lights that are listed.

So a generic link does not really help me.  Id you can directly link to something on this page that can meet the requirements great.

On May 16, 2022, at 2:12 AM, Barbudor ***@***.***> wrote:

﻿

Some are listed here https://templates.blakadder.com/preflashed.html

Their might be more

**Reply by Jason2866** [reply]
So there is probably none.
